networking/kvm.go

```diff
@@ -104,7 +105,7 @@ func setupMacVTapDevice(podID types.UUID, config MacVTapNetConf) (netlink.Link,
 	if config.MTU != 0 {
 		mtu = config.MTU
 	}
-	nameTemplate := fmt.Sprintf("rkt-%s-vtap%%d", podID.String()[0:4])
+	nameTemplate := fmt.Sprintf("rkt-%s-vtap%d", podID.String()[0:4], interface_number)
```

Comment:
It looks like that it used to be some kind of a printf-template before (`rkt-3729-vtap%d`), but now it is not. Should the code expecting it as a template should be updated then (if there is such somewhere)? Also, the `nameTemplate` is misleading - it is not a template anymore, right?